pkg/ui/workspaces/cluster-ui/src/highlightedText/highlightedText.tsx

```diff
@@ -80,18 +80,12 @@ export function getHighlightedText(
     return text;
   }
   highlight = highlight.replace(
-    /[°§%()\[\]{}\\?´`'#|;:+-]+/g,
+    /[°§%()\[\]{}\\?´`'#|;:+-^*]+/g,
```

Comment:
The position of `-` here doesn't seem to be valid. A regexp of the form `[abc-jkl]` includes all characters between `c` and `j`, that is it is equivalent to `[abcdefghijkl]`.

If you want to include `-` in the list, you need to do it in a different way.

Reply:
on the test of the function it works, also when I check manually for the replace:
```
let highlight = "select * from long-name";
highlight = highlight.replace(
    /[°§%()\[\]{}\\?´`'#|;:+-^*]+/g,
    "highlightNotDefined",
  );
  
  
value printed = 'select highlightNotDefined from longhighlightNotDefinedname'
```

Reply:
what if I remove from the list and do a replace just for it, e.g.
```
let a = "abc-def-dert-rtwer-qwe + * ?";
a = a.replace(/[°§%()\[\]{}\\?´`'#|;:+^*]+/g, "X");
'abc-def-dert-rtwer-qwe X X X'
a = a.replace(/[-]/g, "X");
'abcXdefXdertXrtwerXqwe X X X'
```

Reply:
if the regex ends with `-` it uses that literal character, but if the `-` is between two value then it uses the value in between, so I just need to move it back to the "end of the line", like it was previous to me adding the two new ones